### ISSUE TITLE
sonarjs/no-duplicate-string : ignore import expressions

### ISSUE THREAD

**Issue by a-tarasyuk** [SonarSource/eslint-plugin-sonarjs#295]
**I want to report a bug.**

**Reproducer**

```javascript
import("./main");
import("./main");
import("./main");
```

**Expected behavior**
No error 

**eslint-plugin-sonarjs version:** `0.10.0`
**eslint version:** `7.32.0`
**Node.js version:** `14.17.1`
**Rule key:** `no-duplicate-string`

**Comment by vilchik-elena** [comment]
Why would you want to import the same module 3 times?

**Comment by a-tarasyuk** [comment]
@vilchik-elena This is just a simplified example. `no-duplicate-string` ignores import declarations and cjs requires, however, import expressions are not ignored. Does it make sense to ignore them?

https://github.com/SonarSource/eslint-plugin-sonarjs/blob/62503002e9ab118d9b68382be6e80a01f19e33e7/tests/rules/no-duplicate-string.test.ts#L81-L83

https://github.com/SonarSource/eslint-plugin-sonarjs/blob/62503002e9ab118d9b68382be6e80a01f19e33e7/src/rules/no-duplicate-string.ts#L30-L31

--- 

https://github.com/SonarSource/eslint-plugin-sonarjs/blob/62503002e9ab118d9b68382be6e80a01f19e33e7/tests/rules/no-duplicate-string.test.ts#L94-L97

https://github.com/SonarSource/eslint-plugin-sonarjs/blob/62503002e9ab118d9b68382be6e80a01f19e33e7/src/rules/no-duplicate-string.ts#L101
